invoicing/functions/server/slow.js

```diff
@@ -31,5 +32,47 @@ exports.REST = {
                 return resolve({data: {success: true, time}});
             }, time * 1000);
         });
+    },
+
+    $slower: {
```

Comment:
What make this function exposed ? The "scope = public" ?

Reply:
yes